src/Uno.UI.Runtime.Skia.Wpf/Builder/WpfHostBuilder.cs

```diff
@@ -26,6 +26,36 @@ public WpfHostBuilder()
 		}
 	}
 
+	private static void AdjustPath(string dotnetSharedDirectory, string runtimeVersion, ref string windowsDesktopFrameworkPath)
+	{
+		// In at least preview SDKs, the .NET runtime version is different from the .NET Desktop runtime version for the same .NET SDK.
```

Comment:
Interesting change. We may need to adjust this for the net9 release as well.

Reply:
So far it's looking like preview-specific. But we'll see how it goes with stable SDK